saturn-datepicker/src/datepicker/calendar-body.html

```diff
@@ -34,13 +34,15 @@
       [tabindex]="_isActiveCell(rowIndex, colIndex) ? 0 : -1"
       [class.mat-calendar-body-disabled]="!item.enabled"
       [class.mat-calendar-body-active]="_isActiveCell(rowIndex, colIndex)"
-      [class.mat-calendar-body-begin-range]="begin === item.value"
-      [class.mat-calendar-body-end-range]="end === item.value"
-      [class.mat-calendar-cell-semi-selected]="_isSemiSelected(item.value)"
+      [class.mat-calendar-body-begin-range]="_isBegin(item.value)"
+      [class.mat-calendar-body-end-range]="_isEnd(item.value)"
+      [class.mat-calendar-cell-semi-selected]="_isSemiSelected(item.value) || _isBetweenOverAndBegin(item.value)"
+      [class.mat-calendar-cell-over]="_cellOver === item.value"
       [attr.aria-label]="item.ariaLabel"
       [attr.aria-disabled]="!item.enabled || null"
       [attr.aria-selected]="selectedValue === item.value"
       (click)="_cellClicked(item)"
+      (mouseover)="_mouseOverCell(item)"
```

Comment:
What happens if cell was "hovered" by keyboard? (using up/down/left/right arrows)

Reply:
Yes. It's supposed to work, but not working

Reply:
Oh. Indeed this regression was introduce by my previous merge request https://github.com/SaturnTeam/saturn-datepicker/commit/81e26e46650f26e65560b469faa268882a3c5780.
I try to correct it.

Reply:
No. It's not working on demo too.

Reply:
This should fix the regression: https://github.com/SaturnTeam/saturn-datepicker/pull/59
I will now work on the enhancement made by the current merge request to add the preview of range selection also when using keyboard.

Reply:
Also there is problem if `closeAfterSelection=false`. The calendar closes immediately after first press `Enter` key

Reply:
If this is ok with the last commit of https://github.com/SaturnTeam/saturn-datepicker/pull/59, then I added the preview of range selection also with keyboard selection.
See https://github.com/SaturnTeam/saturn-datepicker/pull/57/commits/59e44d022a4d593f2c475a9754af86c75ea2c2a2.